Panels/FlexTabber.js

```diff
@@ -217,7 +217,9 @@ define([
                     tabInfo.get$El().removeClass('AXMFlexTabInActiveHighlight');
                 });
 
-                    tabInfo.attachEventHandlers();
+                tabInfo.attachEventHandlers();
+                Msg.broadcast("FlexTabAdded", tabInfo.tabId);
```

Comment:
I see that on line 216 a slideDown method is triggered which takes 200 ms. For my use case I had to wait until the effect was fully completed to instantiate the tabulator table (since it needed to calculate the height of the panel). Maybe to be certain it might be best to send the message when the slidedown is fully finished?

Reply:
Good point, but in this case it wouldn't work well because the tab is activated before that delay is finished. Therefore, if the "FlexTabAdded" message would be broadcasted after the delay, then the "FlexTabActivated" message will be broadcasted before the "FlexTabAdded" message, which is not the correct order.